app/assets/stylesheets/custom.scss

```diff
@@ -1,29 +1,35 @@
 .home-search {
+  text-align: center;
```

Comment:
Properties should be ordered background, margin, margin-bottom, margin-top, min-height, padding, position, text-align, top, vertical-align, width, z-index